poc/TestOfTestFrameworkByReference/app.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
app.config is evil!! you have to remove it please

Reply:
Committed by mistake. It's removed now.